src/aqua/fluence/worker.aqua

```diff
@@ -18,53 +19,60 @@ data WorkerArgs:
 -- Settings of the worker for the decider
 data WorkerSettings:
     -- Trigger config of the worker
-    worker_config: TriggerConfig
+    config: TriggerConfig
```

Comment:
Sadly, we can't add smth like `#[serde(rename = "name")]`. Maybe you can write about KV field (and init_data) names in comments?

Reply:
Also, it will be easier to find with ctrl-f

Reply:
I don't get what you mean